src/list.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable */
```

Comment:
I would suggest not disabling the linters and fixing the problem manually because linters are there to ensure best practices.